### PR TITLE
RN-192: Upgrade SDK to 3.5 and add Session Replay link sharing screen

### PR DESCRIPTION
**Pull request by AldinaSculac**
#### Updated CS SDK on:

- Classic Implemenatation app
- Expo app 
- Implementation/AdobeAnalytics app

#### Classic Implemenatation app:

- added screen to showcase Session Replay link-sharing option

https://github.com/ContentSquare/react-native-sample-app/assets/45870393/9fd7ffc9-4419-46af-92cc-428a518062dc

### CODE REVIEW

File: ClassicImplementation/src/modules/SessionReplay/views/SessionReplayLinking.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { Button, StyleSheet, Text, View } from 'react-native';
+import { colors, gridUnit } from '../../../constants';
+
+export const SessionReplayLinking = () => {
+  const [SRLinkDescription, setSRLinkDescription] = useState('Click on the button to retrieve the current Session Replay link');
+
+  const getCurrentSessionReplayLink = async () => {
+
+    const link = await Contentsquare.getCurrentSessionReplayLink()
+    console.log(link)
```
Comment (Gguigre) [inline ClassicImplementation/src/modules/SessionReplay/views/SessionReplayLinking.tsx:14]: Remove console.log

Comment (AldinaSculac) [reply ClassicImplementation/src/modules/SessionReplay/views/SessionReplayLinking.tsx:14]: Good point, I forgot to remove it.

Comment (antoine-cottineau) [reply ClassicImplementation/src/modules/SessionReplay/views/SessionReplayLinking.tsx]: esLint would help here as well 👀

Comment (AldinaSculac) [reply ClassicImplementation/src/modules/SessionReplay/views/SessionReplayLinking.tsx]: Should we add esLint as a part of this PR?

**Review by big-toni — APPROVED** [review]

**Pull request merged**